### PR TITLE
Fixes #107

### LINKED ISSUE #107: Add isParser function
Many people do not know about `x instanceof Parsimmon.Parser` as an option, so let's also add `Parsimmon.isParser(x)`.

### PR DESCRIPTION
**Pull request by mcapodici**

### CODE REVIEW

File: src/parsimmon.js [hunk 1]
```diff
@@ -30,8 +30,10 @@
     this._ = action;
   };
 
-  Parsimmon.Parser = Parser;
-
+  Parsimmon.isParser = function (obj) {
```
Comment (wavebeem) [inline src/parsimmon.js]: If you change this to `var isParser = Parsimmon.isParser = function(obj) { ...` then we can use it inside the `assertParser` function, which I think would be nice, if you don't mind changing that too.

File: test/parsimmon.test.js [hunk 2]
```diff
@@ -13,7 +13,19 @@ suite('parser', function() {
...
+  });
 
+  test('Parsimmon.isParser', function() {
+   assert.isFalse(Parsimmon.isParser(undefined));
+   assert.isFalse(Parsimmon.isParser({}));
+   assert.isFalse(Parsimmon.isParser(null));
+   assert.isTrue(Parsimmon.isParser(string('x')));
+   assert.isTrue(Parsimmon.isParser(regex(/[0-9]/)));
+  });
+
```
Comment (wavebeem) [inline test/parsimmon.test.js]: Thanks for adding tests 👍

File: src/parsimmon.js (hunk 1 above)
Comment (mcapodici) [inline src/parsimmon.js]: Thanks, yes it is good to not repeat this. I have pushed a new commit.

**Comment by wavebeem** [comment]
I'm going to need to hold off on this PR for a little bit while I evaluate our FantasyLand support, as I think this will break expectations for FantasyLand users.

**Comment by mcapodici** [comment]
Ok fair enough.

File: src/parsimmon.js [hunk 3]
```diff
@@ -30,8 +30,10 @@
     this._ = action;
   };
 
-  Parsimmon.Parser = Parser;
-
```
Comment (wavebeem) [inline src/parsimmon.js]: Would you mind putting that line back, `Parsimmon.Parser = Parser`? And of course removing the accompanying test. I'm going to keep that export after further consideration. But then I'll gladly merge.

Comment (mcapodici) [inline src/parsimmon.js]: Ok, added it back.

**Comment by wavebeem** [comment]
Thanks again for your work on this.

**Pull request merged**